### ISSUE TITLE
Model reloading for ongoing training

### ISSUE THREAD

**Issue by InfluenceFunctional** [InfluenceFunctional/MXtalTools#19]
Currently if we reload a model for further training, the learning rate and batch size are reset to initial values. 

- reload learning rate and batch size from a given checkpoint when picking it up for further training
- not necessary when only doing evaluation
- give some grace for e.g., if running locally, the batch size may be too big

**Comment by InfluenceFunctional** [comment]
Fixed.
